src/Client/App.fs

```diff
@@ -181,9 +182,10 @@ open Elmish.Debug
 // App
 Program.mkProgram init update view
 |> Program.toNavigable (parseHash pageParser) urlUpdate
-|> Program.withConsoleTrace
-|> Program.withReact "elmish-app"
 #if DEBUG
+|> Program.withConsoleTrace
 |> Program.withDebugger
+|> Program.withHMR
```

Comment:
Didn't we agree to name this `Program.withAllTheHotThingsAvailable`? :wink:

Reply:
look at the commit message here. Might bring inspiration

Reply:
To hot to be true 😆